plugins/inventory/foreman.py

```diff
@@ -85,6 +85,60 @@
         description: Number of hosts per batch that will be retrieved from the Foreman API per individual call
         type: integer
         default: 250
+      foreman:
+        use_reports_api:
```

Comment:
why is that a sub-entry of `foreman` instead of being to-level as all the others?

Reply:
This is the way it was setup in the contrib script to read from config. Motivation here was that users upgrading wouldn't have to change their config files.

Reply:
Okay, yeah, I can see compat being a reason.